website/docs/d/nsxt_alb_controller.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_alb_controller"
+sidebar_current: "docs-vcd-datasource-nsxt-alb-controller"
+description: |-
+  Provides a data source to read NSX-T ALB Controller for Providers. It helps to integrate VMware Cloud Director with
+  NSX-T Advanced Load Balancer deployment. Controller instances are registered with VMware Cloud Director instance.
+  Controller instances serve as a central control plane for the load-balancing services provided by NSX-T Advanced Load
+  Balancer.
+---
+
+# vcd\_nsxt\_alb\_controller
+
+Supported in provider *v3.4+* and VCD 10.2+ with NSX-T and ALB.
+
+Provides a data source to read NSX-T ALB Controller for Providers. It helps to integrate VMware Cloud Director with
+NSX-T Advanced Load Balancer deployment. Controller instances are registered with VMware Cloud Director instance.
+Controller instances serve as a central control plane for the load-balancing services provided by NSX-T Advanced Load
+Balancer.
+
+~> Only `System Administrator` can use this data source.
+
+~> VCD 10.3.0 has a caching bug which prevents listing importable clouds immediately after ALB Controller is created.
```

Comment:
I'm a bit confused. We are in controller data source page, but are warning about importable clouds (same message as in vcd_nsxt_alb_importable_cloud). Maybe we should extend this message by providing some more context why user should care, for example, with a link to the importable clouds data source?

Reply:
I have adjusted and a bit more details

Going down hierarchy one creates:
* resource `vcd_nsxt_alb_controller`
* resource `vcd_nsxt_alb_cloud` <- consumes data source `vcd_nsxt_alb_importable_cloud` to create it


Because Terraform acts quickly this bug occurs when `vcd_nsxt_alb_controller` is created and one tries to find importable cloud immediately by using `vcd_nsxt_alb_importable_cloud`.


Now I have tried to cross-reference in all related objects. `vcd_nsxt_alb_controller`, `vcd_nsxt_alb_cloud` and `vcd_nsxt_alb_importable_cloud` itself.